style.css

```diff
@@ -304,4 +309,382 @@ a {
     justify-content: center;
     margin-left: -42px;
   }
+
+  .chat {
+    display: none;
+  }
+
+  .form-2 {
+    display: none;
+  }
+
+  .log-2 {
+    display: none;
+  }
+
+  .footer-hr-2 {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile {
+    background-image: url(/images/twocirclesheader.svg);
```

Comment:
- [ ] Please add `./` before URL path because your background image not working online maybe it's working locally but online it's not.
You can do it like that: `background-image: url(./images/twocirclesheader.svg);`